File: examples/postcode/random/node/returnARandomPostcodeAndItsInformation.js
const { PostcodeService } = require("m3o/postcode");

// Return a random postcode and its related info
async function returnArandomPostcodeAndItsInformation() {
  let postcodeService = new PostcodeService(process.env.MICRO_API_TOKEN);
  let rsp = await postcodeService.random({});
  console.log(rsp);
}

returnArandomPostcodeAndItsInformation();

File: examples/go/postcode/random/returnARandomPostcodeAndItsInformation.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/postcode"
)

// Return a random postcode and its related info
func ReturnArandomPostcodeAndItsInformation() {
	postcodeService := postcode.NewPostcodeService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := postcodeService.Random(&postcode.RandomRequest{})
	fmt.Println(rsp, err)
}
